runtimes/peregrine/src/lib.rs

```diff
@@ -1081,6 +1096,15 @@ pub type Executive = frame_executive::Executive<
 	(
 		pallet_did_lookup::migrations::CleanupMigration<Runtime>,
 		runtime_common::migrations::RemoveInsecureRandomnessPallet<Runtime>,
+		pallet_did_lookup::migrations::CleanupMigration<Runtime>,
+		runtime_common::migrations::RemoveInsecureRandomnessPallet<Runtime>,
+		attestation::migrations::BalanceMigration<Runtime>,
+		delegation::migrations::BalanceMigration<Runtime>,
+		did::migrations::BalanceMigration<Runtime>,
+		pallet_did_lookup::migrations::BalanceMigration<Runtime>,
+		pallet_web3_names::migrations::BalanceMigration<Runtime>,
+		parachain_staking::migrations::BalanceMigration<Runtime>,
+		public_credentials::migrations::BalanceMigration<Runtime>,
```

Comment:
What about the balances pallet? Doesn't it also have a migration we need to run?

Reply:
No, there are no migration for the balance pallet. There are two substrate related migration which are about the [offences](https://github.com/paritytech/substrate/pull/13936/files) and the [nomination pool](https://github.com/paritytech/substrate/pull/13715) pallet.

Reply:
No, the balance pallet does not have any migrations to run. In the [releases](https://github.com/paritytech/polkadot/releases/tag/v0.9.42), there are two substrate related migration ([nomination pool](https://github.com/paritytech/substrate/pull/13715) and the [offences](https://github.com/paritytech/substrate/pull/13936/files) pallet).